### PR TITLE
Adit issue129

### PR DESCRIPTION
**Pull request by aditpatel01**
### Description
Added docusaurus website folder.

## Screenshot:
image

### CODE REVIEW

**Comment by ankithans** [comment]
@aditpatel01 rebase required with main

**Comment by wtrocki** [comment]
@aditpatel01 no need to worry. We can resolve conflicts here.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
If we could rename `my-website` to `website` that will be cool. We give you full credits for creating website, but folder name can be more generic :)

File: my-website/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'My Site',
```
Comment (wtrocki) [inline my-website/docusaurus.config.js:6]: If I can ask to provide some values here
```suggestion
  title: 'Charmil',
```

File: my-website/docusaurus.config.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  tagline: 'Dinosaurs are cool',
+  url: 'https://your-docusaurus-test-site.com',
```
Comment (wtrocki) [inline my-website/docusaurus.config.js:8]: it will be github pages url

File: my-website/docusaurus.config.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'My Site',
+  tagline: 'Dinosaurs are cool',
```
Comment (wtrocki) [inline my-website/docusaurus.config.js:7]: CLI framework

**Comment by wtrocki** [comment]
Some of the unidraw files can be removed. my-website/static/img/logo.svg  etc.

File: my-website/docusaurus.config.js (hunk 3 above)
Comment (aditpatel01) [reply my-website/docusaurus.config.js:7]: Made the changes. If there are any other changes please let me know

**Pull request merged**